apps/codelab/src/app/admin/feedback/feedback.module.ts

```diff
@@ -1,11 +1,48 @@
+import { AngularFireModule } from '@angular/fire';
+import { AngularFireDatabaseModule } from '@angular/fire/database';
+import { AngularFireAuthModule } from '@angular/fire/auth';
 import { NgModule } from '@angular/core';
+import { RouterModule } from '@angular/router';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { BrowserWindowModule } from '@codelab/browser';
 import { CommonModule } from '@angular/common';
+import { SlidesModule } from '@codelab/slides';
+import { environment } from '../../../environments/environment';
+import { FeedbackMessageTableComponent } from './feedback-message-table/feedback-message-table.component';
+import { MatButtonModule, MatCardModule, MatMenuModule, MatSelectModule, MatTableModule, MatIconModule, MatDatepickerModule, MatNativeDateModule, MatInputModule, MatFormFieldModule, MatExpansionModule } from '@angular/material';
```

Comment:
Can you split imports like:
import {MatButtonModule} from '@angular/material/button';
import {MatCardModule} from '@angular/material/card';

and so on